PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/configuration/ArcadiaSparkSubmissionContentPanel.kt

```diff
@@ -54,6 +55,7 @@ class ArcadiaSparkSubmissionContentPanel (project: Project) : SparkSubmissionCon
             arcadiaData.sparkCompute = cluster.name
             arcadiaData.livyUri = cluster.connectionUrl
                     ?: throw RuntimeConfigurationWarning("Can't get Synapse Spark pool connection URL")
+            arcadiaData.isCosmosCluster = cluster is SynapseCosmosSparkPool
```

Comment:
Could you use a string or enum value for it? Then we needn't to change it next time.